content/departments/product-engineering/process/planning-communication.md

```diff
@@ -0,0 +1,80 @@
+# Product/Engineering planning communication
+
+This page documents how we communicate our plans and progress.
+
+## Annual strategy
+
+Our vision is the story of where we want to be at the end of the year and our strategy describes how we are going to achieve that vision. This is intended to be a stable plan that does not change throughout the year.
+
+[Current vision and strategy.](../strategy-goals/index.md)
+
+## Quarterly department OKRs
+
+Each quarter we set OKRs at the department level (all of product and engineering).
+
+OKRs define what success looks like each quarter, taking into account where we are at as a company and what we think the next things we need to accomplish given the vision and strategy we have for the year.
+
+We don't do OKRs at lower levels (orgs, teams); instead we ask orgs and teams to focus on building out their roadmap (using strategy and OKRs as an input).
```

Comment:
Maybe worth mentioning that this doesn't mean teams are forbidden from setting their own quarterly goals, just that they aren't tracked at the product-engineering level. Feels sort of obvious, but maybe good to be explicit.

Reply:
Does this wording change address your comment?
https://github.com/sourcegraph/handbook/pull/2247/commits/5aad28aa96565441d78663aab2cdd5f77bc85006

Reply:
Yes, thanks!